dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogCustomer/PriceTiersTest.php

```diff
@@ -117,6 +117,40 @@ public function testGetLowestPriceForGuest()
         $this->assertEquals(round(7.25, 2), $this->getValueForQuantity(8, $itemTiers));
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Catalog/_files/simple_product_without_tier_price.php
```

Comment:
Just wondering, why can't we use any existing already fixture that doesn't have tier prices? For example:
- dev/tests/integration/testsuite/Magento/Catalog/_files/second_product_simple.php
- dev/tests/integration/testsuite/Magento/Catalog/_files/product_simple_tax_none.php